index.html

```diff
@@ -436,30 +452,39 @@ <h2><dfn>disconnect()</dfn> method</h2>
     <h2>Processing</h2>
     <section data-link-for="PerformanceObserver">
     <h2>Queue a <code>PerformanceEntry</code></h2>
+      <p><i>Specifications</i> of performance entries may define an algorithm
+        <var>should append to observer</var> which receives a <a>PerformanceEntry</a> and
+        a <a>PerformanceObserver</a> as input and returns a boolean specifying whether the
+        observer should be notified of the entry.
       <p>To <dfn>queue a PerformanceEntry</dfn> (<var>new entry</var>) with an
-      optional <var>add to performance entry buffer</var> flag, which is unset
-      by default, run these steps:</p>
+        optional <var>add to performance entry buffer</var> flag, which is unset by
+        default, run these steps:</p>
```

Comment:
This step should be with respect to before or after onload, not the passed in flag. 

For context, we ruled out the flag approach because it would, once again, require updating every upstream spec to have this on by default.. and I can't think of cases where we'd want this to be off by default for before-onload.

Reply:
Sure, but I haven't added this flag, it's currently in the spec. How about we do a separate PR for this independent change?